syntax = "proto3";

package arithmetic_proto;

message MaxRequest
{
    int64 number = 1;
}

message MaxResponse
{
    int64 max = 1;
}

service MaxService
{
    rpc Max(stream MaxRequest) returns (stream MaxResponse){};
}
